docs/dev/table/sql.md

```diff
@@ -2247,6 +2247,17 @@ TIMESTAMPADD(unit, interval, timestamp)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+TIMESTAMPDIFF(unit, timestamp1, timestamp2)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns the (signed) number of timeUnit intervals between timestamp1 and timestamp2. The unit for the interval is given by the unit argument, which should be one of the following values: <code>SECOND</code>, <code>MINUTE</code>, <code>HOUR</code>, <code>DAY</code>, <code>WEEK</code>, <code>MONTH</code>, <code>QUARTER</code>, or <code>YEAR</code>. E.g. <code>TIMESTAMPDIFF(DAY, DATE '2003-01-02', DATE '2003-01-03')</code> leads to <code>1</code>.</p>
```

Comment:
Could you create a table of time intervals units and time point units? We could move this to a subsection and reference it for all functions.